app/server.R

```diff
@@ -11,10 +11,9 @@ library(lmom) # frequency distribution fitting
 library(plotly) # interactive plots
 library(rlang) # distribution error handling
 library(renv) # package version control
-library(shinybusy) # busy indicator for rendering plots/tables
 library(FlowScreen) # hydrograph and trend test functionality
 library(pastecs) # quick descriptive stats stat.desc()
-library(httr) #http_status() to check HYDAT version
```

Comment:
isn't this still required to go and check the HYDAT version?

Reply:
it is added back now